lib/airbrake/rake/tasks.rb

```diff
@@ -106,8 +106,19 @@
            " environment will be used."
     end
 
+    unless (repo = ENV['REPOSITORY_URL'])
+      repo = `git remote get-url origin`.chomp
+      if !repo.empty?
```

Comment:
Right, but I think it reads better this way with the `else` block containing the message when neither of our first two checks worked.